azure-pipelines.yml

```diff
@@ -43,7 +43,7 @@ jobs:
         parameters:
           fileOriginal: $(CredFile)
           fileLibrary: $(CredFileLibrary)
-
+          
```

Comment:
This looks like an unintentional change. Could we revert the added whitespace?

Reply:
Removed! That was changed when I was trying to figure out why the pipeline was failing